packages/playground/stories/FittedImage.story.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { FittedImage } from '@appearhere/bloom';
+
+storiesOf('FittedImage', module)
+  .add('`object-fit: cover`', () => (
+    <FittedImage
+      style={{
+        backgroundColor: '#eee',
+        objectFit: 'cover',
+        width: '100%',
+        maxWidth: '500px',
+        height: '500px',
+      }}
+      src="https://source.unsplash.com/random/1600x800"
```

Comment:
This might slow it down, but then again it's just for us. I don't know, if you want to use an image from S3, feel free, if not, leave it like this. I'm sure somebody will complain if it's too slow anyway 😄